src/commands/variations/create.ts

```diff
@@ -43,13 +44,14 @@ export default class CreateVariation extends CreateCommand {
         const { headless, key, name, variables } = flags
 
         let featureKey
+        let feature_id
         if (!args.feature) {
             const { feature } = await inquirer.prompt([featurePrompt], {
                 token: this.authToken,
                 projectKey: this.projectKey
             })
-
-            featureKey = feature
+            featureKey = feature.key
+            feature_id = feature._id
```

Comment:
nitpick: should this be in camel case?

Reply:
normally we use snake case for fields that contain mongo ids, but im open to debate on it

Reply:
Though it does have precedent in this repo already to use snake case

Reply:
ah i see, nvm then